changelogs/unreleased/4769-half-life666

```diff
@@ -0,0 +1 @@
+Skip povdolumerestore creation when restore excludes pv/pvc
```

Comment:
Typo: podvolumerestore

Reply:
Corrected

Reply:
Have you address it?

Reply:
Oh, I've corrected the commit message.. not this, let me correct this as well.